contributors/devel/issues.md

```diff
@@ -1 +1,222 @@
+    ```
+    **Action required**: Must specify at most one of `status/in-review` or `status/in-progress`.
+    ```
+  - **priority/important- issues**
+    - The bot includes a warning in the issue comment that the issue will be moved out of the milestone at code freeze.
+    ```
+    **Note**: This issue must be resolved or labeled as priority/critical-urgent by
+    <date of code freeze> or it will automatically be moved out of the milestone
+    ```
+  - **priority/critical-urgent issues**
+    - The bot includes a warning in the issue comment that the issue must be updated regularly.
+      ```
+      **Note**: This issue is marked as priority/critical-urgent, and is expected to be updated at
+      least every 3 days.
+      ```
+    - If an issue hasn't been updated for more than 3 days, the bot comments and adds the `milestone/needs-attention` label.
+      ```
+      **Action Required**: This issue is marked as priority/critical-urgent, but has not been updated
+      in 3 days.  Please provide an update.
+      ```
+    - Owner updates can be a short ACK, but should include an ETA for completion and any risk factors.
+      ```
+      ACK.  In progress
+      ETA: DD/MM/YYYY
+      Risks: Complicated fix required
+      ```
+
+      ```
+      ACK.  In progress
+      ETA: ???
+      Risks: Root cause unknown.
+      ```
+4. Code freeze
+  - **priority/important- issues**
+    - The bot removes non-blocker issues from the milestone, comments as to why this was done, and adds the `milestone/removed` label.
+      ```
+      **Important**: Code freeze is in effect and only issues with priority/critical-urgent may remain
+      in the active milestone.  Removing it from the milestone.
+      ```
+  - **priority/critical-urgent issues**
+    - If an issue has not been updated within 2 days, the bot comments and adds the `milestone/needs-attention` label.
+
+## Escalation
+
+SIGs will have issues where the release team determines that
+additional attention is required.  These issues are escalated through
+the following channels
+
+- Comment mentioning the sig team appropriate for the issue type
+- Emailing the SIG googlegroup list
+  - bootstrapped with the emails addresses from the [community sig list](/sig-list.md)
+  - maybe configured to email alternate googlegroup
+  - maybe configured to directly email SIG leads or other SIG members
+- Messaging the SIG slack channel, mentioning the SIG leads
+  - bootstrapped with the slackchannel and SIG leads from the [community sig list](/sig-list.md)
+  - maybe configured to message alternate slack channel and users
+
+## Issues tracked in other repos
+
+Some issues are filed in repos outside the [kubernetes repo].  The bot must also be run against these
+repos and follow the same pattern.  The release team can query issues across repos in the kubernetes org using
+a query like [this](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.7+org%3Akubernetes+)
```

Comment:
let's update sample link to a more recent version

Reply:
tweaked to 1.12